Client/Frontend/TabContentsScripts/CreditCardHelper.swift

```diff
@@ -8,28 +8,31 @@ import WebKit
 import Common
 import Storage
 
-enum ValidFieldType: String, CaseIterable {
-    case ccNumber = "cc-number"
-    case ccExpMonth = "cc-exp-month"
-    case ccExpYear = "cc-exp-year"
-    case ccName = "cc-name"
-}
-
 struct Payload: Codable {
```

Comment:
Would something like `CreditCardPayload` be a better name for this? Then you don't have to prefix with `cc` for stuff, and it reads more straightforward? Plus, I'm seeing a bunch of other things with `CreditCard` prefix in the title

Reply:
Good call, changed name to `CreditCardPayload`